sample_project/input_config.yaml

```diff
@@ -1,5 +1,18 @@
 videos_dir_path: ./sample_project/videos
 metadata_fields_file_path: ./sample_project/metadata_fields.yaml
 metadata_key_field_str: File
-pose_estimation_model_str: DLC_resnet50_jwasp_femaleandmaleSep12shuffle1_1000000
 pose_estimation_results_path: ./sample_project/pose_estimation_results
+pose_estimation_model_str: DLC_resnet50_jwasp_femaleandmaleSep12shuffle1_1000000
+dashboard_export_data_path: ./sample_project/output
+roi_names:
+  - enclosure
+  - left_box
+  - top_box
+  - right_box
+  - bottom_box
+  - tube
+event_tags:
+  - experiment_start
+  - box_1_open
+  - box_1_closed
+  - experiment_end
```

Comment:
Probably not a concern yet, but we should bear in mind that the format of the input config file (and probably some of our other YAMLs) will be tied to versions of our code.

So once we're out of beta, we're _potentially_ going to need to support our users transitioning, for example, an old project with an old input config format according to anything we've changed.

Reply:
Not sure if this is worth a ticket... ?